File: hw/tipan/u1/hw07Collections/Collection.py/Collections.py
from datetime import date

class Person:

    def __init__(self, id, full_name, born_on_date, alive):
        self.id = id
        self.full_name = full_name
        self.born_on_date = born_on_date
        self.alive = alive

    def __str__(self):
        return f"{self.id} - {self.full_name} - {self.born_on_date} - {self.alive}"


person1 = Person(
    1,
    "Ronald",
    date(2005, 5, 10),
    True
)

person2 = Person(
    2,
    "Valeria",
    date(2006, 8, 15),
    True
)

people = []

people.append(person1)
people.append(person2)

for p in people:
    print(p)

File: my-fresh-copy/ws/tipan/u1/ws08Collections/Collection.js/Collections.js
class Person {

    constructor(id, fullName, bornOnDate, alive) {
        this.id = id;
        this.fullName = fullName;
        this.bornOnDate = bornOnDate;
        this.alive = alive;
    }

    toString() {
        return `${this.id} - ${this.fullName} - ${this.bornOnDate} - ${this.alive}`;
    }
}

const person1 = new Person(
    1,
    "Ronald",
    "2005-05-10",
    true
);

const person2 = new Person(
    2,
    "Valeria",
    "2006-08-15",
    true
);

const people = [];

people.push(person1);
people.push(person2);

people.forEach((p) => {
    console.log(p.toString());
});